cluster-autoscaler/simulator/cluster.go

```diff
@@ -257,3 +273,7 @@ func shuffleNodes(nodes []*apiv1.Node) []*apiv1.Node {
 	}
 	return result
 }
+
+func hasNoScaleDownAnnotation(node *apiv1.Node) bool {
+	return node.Annotations[ScaleDownDisabledKey] == "true"
```

Comment:
I can change that if necessary, but Golang documentation for maps specifies that nil map behaves like an empty map on read, so this function works as expected.

https://blog.golang.org/go-maps-in-action